tests/json-fixtures/test_blockchain.py

```diff
@@ -34,6 +36,84 @@
 BASE_FIXTURE_PATH = os.path.join(ROOT_PROJECT_DIR, 'fixtures', 'BlockchainTests')
 
 
+# These are the slowest tests from the full blockchain test run. This list
+# should be regenerated occasionally using `--durations 100`.
+SLOWEST_TESTS = {
```

Comment:
Not for this PR but just as an idea: Isn't there something in Circle CI that we can differentiate between CI runs for PRs and CI runs on master? I think it would be nice if we could at least achieve that all these tests run from time to time. So that, even if we don't get immediate feedback that our PR broke one of these, we at least get notified about it some time after.

Reply:
It's possible with Travis and I would be surprised if it wasn't also possible with Circle CI. It could be nice to have a nightly build which takes master and runs all the tests, and maybe even does benchmarking?

Reply:
I dug around a bit and couldn't find anything in the CircleCI interface to do this

Reply:
CircleCI exposes [some useful environment variables](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables) such as CIRCLE_PULL_REQUEST and CIRCLE_BRANCH you could use to detect whether you're building a branch or not.

I've [opened a PR](https://github.com/ethereum/py-evm/pull/1588) which attempts to schedule a nightly build which runs all the tests, even the slowest ones.